USER_STORIES.md

```diff
@@ -0,0 +1,122 @@
+# User Stories
+
+The following user stories should be runnable without error in production.
+
+## Template
+
+**As a:** (Describe the user and set out the starting setup or application state)
+
+**want to:** (Describe what the user wants to do)
+
+**that:** (Describe why the user wnats to do it)
+
+**When I Do => I Get:** (Describe the action(s) taken by the user and the feedback they get with each step to know that they are successful.)
+
+## Story 1
+
+**As an:** Anonymous first time user, working on a desktop computer running Chrome. I don't fully trust the website and so I want to sign in with a fake email account and NOT with Google.
+
+**I want to:** Log in to participate in the competition.
+
+**So that:** I can see my opinion posted on the application.
+
+**When I Do => I Get:**
+
+1. Navigate to https://celebrity-fanalyzer.pages.dev/
+2. Navigate to the profile page and choose `SIGN UP`
+3. Navigate to the `ADMIN` panel.
```

Comment:
The user cannot navigate to `/admin` after signup.
He is currently required to be "promoted" to Writer.